client/src/app/App.js

```diff
@@ -118,8 +118,14 @@ export class App extends PureComponent {
 
     const userPlugins = this.getPlugins('client');
 
+    let defaultPluginsAry = this.props.hasServerInteraction ? defaultPlugins : (
```

Comment:
What does _Ary_ stand fore here? This should also be a `const` I guess.

Reply:
This is now `const selectedDefaultPlugins`